Sources/SwiftDocCUtilities/TemplateLibrary/Init/___FILEBASENAME_INIT___.docc/Documentation.md

```diff
@@ -0,0 +1,33 @@
+# Documentation
+
+@Metadata {
+  @TechnologyRoot
+}
+
+<!--@START_MENU_TOKEN@-->Summary<!--@END_MENU_TOKEN@-->
+
+## Overview
+
+<!--@START_MENU_TOKEN@-->Overview<!--@END_MENU_TOKEN@-->
+
+@Image(source: "DocC.png", alt: "An illustration of the DocC logo.")
+
+## Usage Instructions
+
+To start previewing this documentation run:
+```
+docc preview ___CATALOG_OUTPUT_PATH___
```

Comment:
What would `CATLOG_OUTPUT_PATH` end up being expanded to? Is it an absolute path? My concern with that is as soon as they move this catalog to a different location on their file system it will stop making sense.

Maybe we could do something like this?

```
To preview this documentation, use your terminal to navigate to the root of this
DocC catalog and run:

docc preview
```

Reply:
It was an absolute path initially, but I changed it for your suggestion in https://github.com/apple/swift-docc/pull/719/commits/2a993c4e59d8222a3359af61a8b4ef0b3be9bdcf, thanks Ethan! 😄